app/src/conferences/forms.py

```diff
@@ -60,9 +53,6 @@ class Meta(UserPreferencesWithOrgForm.Meta):
         model = UserPreferences
         exclude = ['user', 'zosia', 'payment_accepted', 'bonus_minutes']
 
-    # Yes, it's required by default. But it's insane - better be verbose than misunderstood.
-    accepted = forms.BooleanField(required=True)
```

Comment:
So it is True by default now?

Reply:
This field wasn't even written to model! I suppose it should've been used to accept terms and conditions here: https://github.com/ksiuwr/zosia16-site/blob/f4bb31ecef26bd73f9a3e7919011cd7d50c6ec79/app/src/conferences/templates/conferences/register.html#L27

Reply:
Now, on second thought, I think that we should restore this and put here accepting of the ZOSIA T&C and at the same time remove it from the first registration form. What do you think about this?

Reply:
Done